telosnetwork/telos-wallet
# #441 | Wrap/unwrap pages

# Fixes #441 

## Description
This PR adds a new layout for displaying sidebar text (such as on the wrap/unwrap/staking/unstaking pages), and adds it onto the wrap/unwrap pages

## Test scenarios
- this PR must be tested locally because its base branch is not `develop`
- from the project root, run `git fetch --all && git checkout 441-wrap-page-sidebar && yarn install && yarn dev`
- go to http://localhost:8081
- log in
- click the wrap link on the nav
  - the tabs should match the [designs](https://www.figma.com/file/eIpbnt6vqwqptEbZWtwO8Y/v0.3?type=design&node-id=703-5046&mode=design&t=wmsRDyLz8pkfEVYG-0)
- go to the Wrap tab and wrap any amount of TLOS
  - You should be receiving a neutral notification of the transaction sent and later a success notification with the tx hash.
  - The balances for TLOS and WTLOS should be updated automatically, that includes:
    - the headers balances
    - the max available for wrap
- go to the Unwrap tab and unwrap any amount of WTLOS
  - You should be receiving a neutral notification of the transaction sent and later a success notification with the tx hash.
  - The balances for TLOS and WTLOS should be updated automatically, that includes:
    - the headers balances
    - the max available for unwrap
- Repeat the previous process with all different login methods (OreId, SafePal, MetaMask, WalletConnect)


## Checklist:
